roles/gitea/defaults/main.yml

```diff
@@ -85,7 +85,28 @@ gitea_mailer_enabled: no
 # Mail settings below are required if gitea mailer is enabled
 # 'From' address used in mails sent by gitea
 # gitea_mail_from: "gitea-noreply@{{ gitea_fqdn }}"
-# SMTP host, username and password for outgoing mail from gitea
-# gitea_mail_host: "CHANGEME"
+
 # gitea_mail_user: "CHANGEME"
 # gitea_mail_password: "CHANGEME"
+
+# New in 1.18.0 :
+
+# Mail server protocol. One of "smtp", "smtps", "smtp+starttls", "smtp+unix", "sendmail", "dummy".
+# - sendmail: use the operating system's `sendmail` command instead of SMTP. This is common on Linux systems.
+# - dummy: send email messages to the log as a testing phase.
+# If your provider does not explicitly say which protocol it uses but does provide a port,
+# you can set SMTP_PORT (gitea_mail_smtp_port) instead and this will be inferred.
+# gitea_mail_protocol: "CHANGEME"
+
+# Mail server address, e.g. smtp.gmail.com.
+# For smtp+unix, this should be a path to a unix socket instead.
+# (Before 1.18, see the notice, this was combined with SMTP_PORT as HOST.)
+# gitea_mail_smtp_addr: "CHANGEME"
+
+# Mail server port. Common ports are:
+#   25:  insecure SMTP
+#   465: SMTP Secure
+#   587: StartTLS
+# If no protocol is specified, it will be inferred by this setting.
+# (Before 1.18, this was combined with SMTP_ADDR as HOST.)
+# gitea_mail_smtp_port: "CHANGEME"
```

Comment:
Let's keep it short for consistency with other roles:

```suggestion
# Mail server address, e.g. smtp.gmail.com.
# gitea_mail_host: "CHANGEME"
# Mail server protocol (smtp/smtps/smtp+starttls/smtp+unix/sendmail/dummy)
# gitea_mail_protocol: "CHANGEME"
# Mail server port (e.g. 25/465/587)
# gitea_mail_smtp_port: "CHANGEME"
```

Reply:
`gitea_mail_host` is no more needed. I will add a commit to fix that.